doc/manual/21-preliminary_steps.en.md

```diff
@@ -29,24 +29,24 @@ postgres@pg$ createuser -P barman
 ```
 
 ``` sql
-GRANT EXECUTE ON FUNCTION pg_start_backup(text, boolean, boolean) to barman;
-GRANT EXECUTE ON FUNCTION pg_stop_backup() to barman;
-GRANT EXECUTE ON FUNCTION pg_stop_backup(boolean, boolean) to barman;
+GRANT EXECUTE ON FUNCTION pg_backup_start(text, boolean) to barman;
+GRANT EXECUTE ON FUNCTION pg_backup_stop(boolean) to barman;
 GRANT EXECUTE ON FUNCTION pg_switch_wal() to barman;
 GRANT EXECUTE ON FUNCTION pg_create_restore_point(text) to barman;
 
 GRANT pg_read_all_settings TO barman;
 GRANT pg_read_all_stats TO barman;
 ```
 
-In the PostgreSQL 15 beta and any subsequent PostgreSQL versions the functions
-`pg_start_backup` and `pg_stop_backup` have been renamed and have different
-signatures. You will therefore need to replace the first three lines in the
+In the case of using PostgreSQL version 14 or a prior version, the functions
+`pg_backup_start` and `pg_backup_stop` had different names and different
+signatures. You will therefore need to replace the first two lines in the
```

Comment:
putting myself in the user shoes, i think I would love to see here a link to the old documentation functions...
I don't know in general I think we should add more links to pg doc when we put code blocks like this one. maybe is not a suggestion for now...

Reply:
I agree, but I didn't want to make this PR about refactoring the docs. Maybe we should track that separately